<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="QA-2-Win8_1-64" name="com.automation.main.TC18836ValidateTheSourceTypeAsRecordingTextInSearchFieldOnTheCourseLevel" tests="2" failures="1" timestamp="6 Sep 2016 11:18:36 GMT" time="16.019" errors="0">
  <testcase name="loginCourses" time="11.019" classname="com.automation.main.TC18836ValidateTheSourceTypeAsRecordingTextInSearchFieldOnTheCourseLevel">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at java.lang.String.startsWith(Unknown Source)
	at java.lang.String.startsWith(Unknown Source)
	at com.automation.main.CoursesHelperPage.selectCourseThatStartingWith(CoursesHelperPage.java:406)
	at com.automation.main.TC18836ValidateTheSourceTypeAsRecordingTextInSearchFieldOnTheCourseLevel.loginCourses(TC18836ValidateTheSourceTypeAsRecordingTextInSearchFieldOnTheCourseLevel.java:195)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- loginCourses -->
  <testcase name="loadPage" time="5.000" classname="com.automation.main.TC18836ValidateTheSourceTypeAsRecordingTextInSearchFieldOnTheCourseLevel"/>
</testsuite> <!-- com.automation.main.TC18836ValidateTheSourceTypeAsRecordingTextInSearchFieldOnTheCourseLevel -->
